doc/releases/changelog-dev.md

```diff
@@ -153,6 +153,9 @@
 
 <h3>Improvements</h3>
 
+* `qml.ctrl` now uses `Controlled` instead of `ControlledOperation`.  The new `Controlled` class
+  wraps individual `Operator`'s instead of a tape.  It provides improved representations and integration.
```

Comment:
```suggestion
  wraps individual `Operator`'s instead of a tape.  It provides improved representations and integration.
  [(#2990)](https://github.com/PennyLaneAI/pennylane/pull/2990)
```